js/SearchResult.js

```diff
@@ -0,0 +1,47 @@
+import {fetchData} from "./utils.js"
+const URLForNameAndSymbol = `https://stock-exchange-dot-full-stack-course-services.ew.r.appspot.com/api/v3/search?query=`
+const QuaryURLForNameAndSymbol = `&limit=10&exchange=NASDAQ`;
+
+
+export class SearchResults {
+    constructor (companies){
+        this.companies = companies;
+    }
+
+    async renderResults() {
+        this.companies.innerHTML = "";
+        const companyResultResponse = await fetchData(URLForNameAndSymbol + document.getElementById('companySearch').value + QuaryURLForNameAndSymbol);
+        const unOrderedList = document.createElement("ul");
+        const numOfResluts = Math.min(companyResultResponse.length, 10);
+
+        for (let i = 0; i < numOfResluts; i++) {
+          let symbol = companyResultResponse[i].symbol;
```

Comment:
Is the symbol is going to change through this loop? 
If not, it should be a const.